src/node.h

```diff
@@ -201,10 +203,14 @@ NODE_EXTERN extern bool force_fips_crypto;
 #endif
 
 NODE_EXTERN int Start(int argc, char *argv[]);
-NODE_EXTERN void Init(int* argc,
-                      const char** argv,
-                      int* exec_argc,
-                      const char*** exec_argv);
+
+struct ProcessArguments {
+  int argc;
+  char **argv;
```

Comment:
nit: common style here is left-leaning/C++-style pointers, i.e. `char** argv`

Reply:
It's really in my muscle memory to type it like that. I thought I fixed all of those instances, but this one fell through the cracks. Thanks for pointing it out!